### ISSUE TITLE
Jittery scrollbar

### ISSUE THREAD

**Issue by prakharb5** [iaarchiver/MinimalScrollbar#19]
On some websites, the scrolling is smooth, but the scrollbar moves with a lot of jitter. I hope it would be fixed.

**Comment by prakhar-pal** [comment]
Can you name one of those websites?

**Comment by prakharb5** [comment]
Almost any website which has a lot of scrollable content. The scrollbar starts stuttering when I scroll.
If you want a specific website, try https://en.wikipedia.org/wiki/Main_Page 

BTW, I have switched to using this : https://github.com/StylishThemes/Overlay-Scrollbars
Works pretty reliably and is highly customisable.

**Comment by prakhar-pal** [comment]
Hmm, I did notice the same on vlc download page after I commented here. Since then I had created a small chrome extension that replicated macOS style scrollbars. But today I found out that Chrome and it's derivatives support overlay scrollbars out of the box, so I'm using that and those seem promising.

**Comment by prakharb5** [comment]
Overlay scrollbars? Weren't those removed in the recent releases of Chrome? I don't see it in the flags anymore.

**Comment by prakhar-pal** [comment]
@Cyberdroid1 nope, I'm using that overlay-scrollbars on latest versions of chromium and brave

![screeshot](https://i.imgur.com/evvqHSU.png)

**Comment by prakharb5** [comment]
Oh, interesting. I dont find this flag on Microsoft Edge or Chrome anymore.
